lib/index.js

```diff
@@ -25,10 +25,14 @@ const register = (server, options = {}) => {
       {
         assign: 'saml',
         method: async (request, h) => {
-          const SAMLOptions = await getSAMLOptions(request)
+          const { SAMLOptions, redirectUrl } = (await getSAMLOptions(request)) ?? {}
```

Comment:
1. Please adjust the tests + add a test for when there is a redirectUrl
2. Update the README.me to account for this change
3. We need to bump the package.json version ourselves